### DISCUSSION TITLE
change the color of the bookmark bar

### DISCUSSION THREAD

**Discussion by ScZhen** [sumatrapdfreader/sumatrapdf#4194] [General]
Hi everyone, I'm a new user of the program. How can I change the color of the bookmark bar?

**Comment by GitHubRulesOK** [comment]
The Favorites and Bookmarks sidebar is all one with one overall control so can only change colour via the theme options.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/a89cd118-91ce-4a57-a3b5-6b14b208f635)

**Reply by ScZhen** [reply]
Thanks.

**Comment by GreatKin9** [comment]
The background color of main window has been changed to green.
I want to change the background color of bookmark, where surrounded by golden line in the attachment. 
Maybe I will have the same answer? But hope it can be divided into two sections for color changing separately.
![无标题](https://github.com/user-attachments/assets/c47493dd-7799-485d-a023-6b7925a321d8)

**Reply by GitHubRulesOK** [reply]
![image](https://github.com/user-attachments/assets/876b00db-7be9-4d53-8d78-ecda5e575c01)

**Reply by GreatKin9** [reply]
Maybe we have different setting.txt. I can change the background color of main window only, but can't change the background color of bookmark like you.
![无标题](https://github.com/user-attachments/assets/0e972633-6c0c-417e-9d96-643fd7bd6a72)

**Reply by GitHubRulesOK** [reply]
your settings are older layout  and  some versions were not in sync between main window and bookmarks.

As only current pre-release can be changed for any problems I tested Latest pre-release which uses the newer alphabetic order
[https://www.sumatrapdfreader.org/prerelease](https://www.sumatrapdfreader.org/prerelease)

**Reply by GreatKin9** [reply]
Thank you so much.
